assets/js/modules/reader-revenue-manager/components/settings/SettingsView.js

```diff
@@ -71,7 +71,7 @@ export default function SettingsView() {
 	} );
 
 	return (
-		<div className="googlesitekit-setup-module googlesitekit-setup-module--reader-revenue-manager">
+		<div className="googlesitekit-setup-module googlesitekit-setup-module-view googlesitekit-setup-module--reader-revenue-manager">
```

Comment:
This additional class name should not be needed as we should be able to determine the screen based on already available selectors.